app/views/admin/statistics_announcement_unpublishings/new.html.erb

```diff
@@ -0,0 +1,48 @@
+<% content_for :page_title, "Unpublish statistics announcement" %>
+<% content_for :context, @statistics_announcement.title %>
+<% content_for :title, "Unpublish statistics announcement" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @statistics_announcement)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-body-lead">
+      <p>Take announcements off GOV.UK if they:</p>
+      <ul>
+        <li>are a duplicate of another announcement</li>
+        <li>have been published too soon or in error</li>
+      </ul>
+    </div>
+
+    <%= render "govuk_publishing_components/components/inset_text", {
+      text: sanitize("If the statistics are no longer being released, #{link_to("cancel the release", [:cancel, :admin, @statistics_announcement], class: "govuk-link")}.")
+    } %>
```

Comment:
i know it's strictly not like for like but showing this when a document is already cancelled feels a bit silly to me. Maybe we can only show this if that isn't the case? wdyt?

Reply:
Agree, it was a pretty straight forward change, so added the behaviour with a test.